src/client_scripts/error/errorHandling.ts

```diff
@@ -0,0 +1,34 @@
+
+interface Window {
+    debug: Function;
+}
+
+interface ErrorInfo {
+    name: string | undefined;
+    message: string | undefined;
+    stack: string | undefined;
+}
+
+const debug = window.debug;
+
+const error = debug('team-link:error');
+
+const sendError = (err: ErrorInfo): void => {
+    fetch('clientError', {
+        method:  'POST',
+        headers: {
+            'Content-Type': 'application/json'
+        },
+        body: JSON.stringify(err)
+    });
+};
+
+window.onerror = (message: string | Event, source: string | undefined, lineno: number | undefined, colno: number | undefined, err: Error | undefined) => {
```

Comment:
Since we do not use some arguments, we can leave them as `unknown`.

Reply:
Also, it's better to use optional parameters (`param?: type`) instead of `param: type | undefined`.